spec.html

```diff
@@ -116,7 +116,7 @@ <h1>WrappedFunctionCreate ( _callerRealm_, _targetFunction_ )</h1>
 			1. Assert: IsCallable(_targetFunction_) is *true*.
 			1. Let _internalSlotsList_ be the internal slots listed in <emu-xref href="#table-internal-slots-of-wrapped-function-exotic-objects"></emu-xref>, plus [[Prototype]] and [[Extensible]].
 			1. Let _obj_ be ! MakeBasicObject(_internalSlotsList_).
-			1. Set _obj_.[[Prototype]] to %Function.prototype% from _callerRealm_.
+			1. Set _obj_.[[Prototype]] to _callerRealm_.[[Intrinsics]].[[*"%Function.prototype%"*]].
```

Comment:
@leobalter I don't think that's how 262 does it today, not need to use quotes or `*`, see `AddRestrictedFunctionProperties` algo.

Reply:
That's correct. I'll roll it back and merge as the original.

Reply:
hm, that seems like an oversight to me. i'll pursue updating it in 262.

Reply:
actually hm, i can see the logic here. since `.[[foo]]` is the string "foo" already, then `.[[%Function.prototype%]]` is the string "%Function.prototype%". thanks for finding that use case.

Reply:
@ljharb this syntax is already used in 262, see the following example:

```
<emu-clause id="sec-addrestrictedfunctionproperties" aoid="AddRestrictedFunctionProperties">
      <h1>AddRestrictedFunctionProperties ( _F_, _realm_ )</h1>
      <p>The abstract operation AddRestrictedFunctionProperties takes arguments _F_ (a function object) and _realm_ (a Realm Record). It performs the following steps when called:</p>
      <emu-alg>
        1. Assert: _realm_.[[Intrinsics]].[[%ThrowTypeError%]] exists and has been initialized.
```